### PR TITLE
Port react-map-gl draw control to @nebula.gl/react-editor-lite

### PR DESCRIPTION
**Pull request by xintongxia**
svg-draw

### CODE REVIEW

File: examples/svg/src/app.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+/* global window */
+import React, {Component} from 'react';
```
Comment (georgios-uber) [inline examples/svg/src/app.js]: NIT: the other examples don't have `src` folder.

File: examples/webpack.config.local.js [hunk 2]
```diff
@@ -39,6 +40,9 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
         '@nebula.gl/overlays/dist': OVERLAYS_SRC_DIR,
         '@nebula.gl/overlays': OVERLAYS_SRC_DIR,
 
+        '@nebula.gl/svg/dist': SVG_SRC_DIR,
+        '@nebula.gl/svg': SVG_SRC_DIR,
```
Comment (georgios-uber) [inline examples/webpack.config.local.js]: Great

File: modules/svg/test/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+// TODO add tests
```
Comment (georgios-uber) [inline modules/svg/test/index.js]: no need for index, just name your files xyz.test.js and jest will auto detect them.

**Comment by georgios-uber** [comment]
I will add a flow ignore rule for now.

**Comment by georgios-uber** [comment]
@xintongxia Can you please add unit tests?

**Comment by georgios-uber** [comment]
How about `react-svg`

**Comment by ibgreen** [comment]
> How about react-svg

I am not sure "svg" says much to the typical user. 

Most users of this class will be looking for a lightweight editor component, they probably do not care much about whether it is using SVG, canvas etc.

Maybe `react-micro-editor`, `react-editor-lite` (or `react-svg-editor` if you really want to see that it renders using SVG).

If you are going for hyper-modularization with multiple react modules, maybe change the name of the other module to `react-overlays`?

**Comment by georgios-uber** [comment]
`react-editor-lite`

**Pull request merged**